x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -238,6 +247,20 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
         <EuiFlexItem grow={false}>
           <span>{getTaskStateBadge(jobStatus)}</span>
         </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiSpacer />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            target="_blank"
+            iconType="help"
+            iconSide="left"
+            color="primary"
+            href={`https://www.elastic.co/guide/en/machine-learning/${metadata.branch}/ml-dfanalytics-evaluate.html#ml-dfanalytics-regression-evaluation`}
+          >
+            Regression evaluation docs
```

Comment:
This text needs internationalizing.

Reply:
Added in c34a7e417ae3143a3124da34460a54398eecc651